bindata/network/multus/multus.yaml

```diff
@@ -245,7 +270,7 @@ spec:
           value: "/usr/src/egress-router-cni/bin/"
       - name: cni-plugins
         image: {{.CNIPluginsImage}}
-        command: ["/entrypoint/cnibincopy.sh"]
+        command: ["/bin/bash", "-c", "/entrypoint/cnibincopy.sh && /entrypoint/allowlistcopy.sh"]
```

Comment:
is `allowlistcopy.sh` really that useful? Couldn't you just `cp /entrypoint/allowlist.conf /host/etc/cni/tuning/`?